server.js

```diff
@@ -1,5 +1,19 @@
-const app = require('./app')
+const app = require("./app");
+const mongoose = require("mongoose");
+require("dotenv").config();
 
-app.listen(3000, () => {
-  console.log("Server running. Use our API on port: 3000")
-})
+const uriDb = process.env.DB_HOST;
+
+const connection = mongoose.connect(uriDb);
+
+connection
+  .then(() => {
+    app.listen(3000, () => {
```

Comment:
```suggestion
    app.listen(port, () => `app running on port ${port}`)
```

Reply:
Port też warto żeby był gdzieś zdefiniowany wyżej ewentualnie też może być w pliku .env bo wtedy można napisać coś takiego jak napisałem żeby dać informację na którym porcie to działa i nie zmienia się dwa razy.  :)